--- sudoku-savant-1.1.orig/sudoku-savant.desktop.in	2006-07-11 01:46:09.000000000 +0200
+++ sudoku-savant-1.1/sudoku-savant.desktop.in	2006-07-25 03:25:56.844731750 +0200
@@ -3,7 +3,10 @@
 Encoding=UTF-8
 Name=Sudoku Savant
 Exec=sudoku-savant
+Icon=sudoku-savant.png
 Terminal=false
-Categories=GNOME;GTK;Application;Game;
+Categories=GTK;Game;BoardGame;
 Comment=Generate and solve Sudoku puzzles
 Comment[de]=Erzeugen und Lösen von Sudokus
+Comment[pl]=Twórz i rozwiązuj puzzle Sudoku
+# vi: encoding=utf-8
