--- sudoku-savant-1.1.orig/src/document.h	2006-07-17 21:13:24.000000000 +0200
+++ sudoku-savant-1.1/src/document.h	2006-07-25 06:32:03.790622250 +0200
@@ -58,6 +58,7 @@
     public:
       Note();
       bool __attribute__((pure)) hasMark(Note::size_type i) const { return (m_mark[i] != 0); } 
+      bool __attribute__((pure)) hasMark(const Square&, size_type) const;
       bool __attribute__((pure)) hasMarks() const { return (m_marks != 0); }
       bool setMark(size_type, unsigned);
       bool clearMark(size_type);
